dss_metrics/minio_rest_collector.py

```diff
@@ -138,7 +142,9 @@ def get_minio_metrics_from_endpoint(self, endpoint):
         r = requests.get(url)
         metrics_data = []
         for line in r.text.splitlines():
-            if (any(line.startswith(metric) for metric in self.minio_metrics)):
+            if (line.startswith("minio")
```

Comment:
redundant paranthesis and make it into a single line like
`if line.startswith('minio') and ....`

Reply:
addressed